### ISSUE TITLE
AdamW Full Implementation Available

### ISSUE THREAD

**Issue by OverLordGoldDragon** [CyberZHG/keras-lookahead#7]
The latest [Lookahead optimizer](https://arxiv.org/abs/1907.08610) paper, co-authored by Geoffrey Hinton, used [AdamW](https://arxiv.org/abs/1711.05101) as its base optimizer, and noted it performing superior to plain Adam. To the best of my knowledge, no complete implementation of AdamW in Keras existed - until now, by me:

[Keras AdamW](https://github.com/OverLordGoldDragon/keras-adamw)

It includes NadamW and SGDW, and their WR (Warm Restart) counterparts - with cosine annealing learning rate schedule, and _per layer_ learning rate multipliers (useful for pretraining). All optimizers are well-tested, and for me have yielded **3-4% F1-score improvements** in already-tuned models for timeseries classification. Up to date with Keras 2.3.0, and can be used with this repo's Lookahead.
